ejb-security-context-propagation/README.adoc

```diff
@@ -195,20 +195,12 @@ Note that the `http-connector` in the `remoting` subsystem uses this `applicatio
 // Build and Deploy the Quickstart JAR
 include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
 
-== Run the Client
-
-Before you run the client, make sure you have already successfully deployed the EJBs to the server in the previous step and that your terminal is still in the same folder.
-
-Type this command to execute the client:
-
-[source,options="nowrap"]
-----
-$ mvn exec:exec
-----
+// Server Distribution Testing
```

Comment:
I simply replaced the exec:exec instructions with the instructions running the integration tests, which you were missing